### PR TITLE
Updated activity_main.xml

### PR DESCRIPTION
**Pull request by parjanyaacoder**
Created a basic layout of the app.Fixes issue#2

### CODE REVIEW

File: activity_main.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (Bhavnaharitsa) [inline activity_main.xml:2]: Constraint layout will cause problems from phone to phone

Comment (parjanyaacoder) [reply activity_main.xml:2]: Should I use linear or relative layout

Comment (parjanyaacoder) [reply activity_main.xml:2]: Yes I Know but please merge this pull request,And you can open this issue after pull request is merged.I tried changing from Constraint Layout to Linear or Relative Layout but thats not working.So please merge this pull  request.

Comment (Bhavnaharitsa) [reply activity_main.xml:2]: How come that wont work? I haved used Linear Layout in all the apps I have built..it works like charm.
Check it and this petty thing cant be opened as an issue later on

**Pull request closed without merging**